massa-consensus-worker/src/consensus_worker.rs

```diff
@@ -1103,7 +1103,27 @@ impl ConsensusWorker {
                     "consensus.consensus_worker.process_protocol_event.get_blocks",
                     { "list": list }
                 );
-                // TODO: use shared storage.
```

Comment:
I tried to follow every branch of this code including message send and receive through the network and didn't see a store or retrieve call. Am I missing something ?

Reply:
I think the store is done in network in this PR: https://github.com/massalabs/massa/pull/2305

retrieve is done in various places, for example in `block_graph` in this PR https://github.com/massalabs/massa/pull/2233